apps/host/src/pages/notice-create/notice-form.tsx

```
@@ -0,0 +1,161 @@
+          <CheckButton checked={isPinned} onChange={handlePinToggle} />
+        </div>
+
+        <InputLayout label='공지 이미지' isEssential={false}>
+          <div
+            ref={scrollRef}
+            className={styles.imageListContainer}
+            onMouseDown={onDragStart}
+            onMouseMove={onDragMove}
+            onMouseUp={onDragEnd}
+            onMouseLeave={onDragEnd}
+          >
```

Biome (2.4.6)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)